subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c

```diff
@@ -59,9 +59,9 @@ static struct {
 } event;
 
 /* Entropy device */
-#if defined(CONFIG_ENTROPY_NRF5_RNG)
+#if defined(CONFIG_ENTROPY_HAS_DRIVER)
 static const struct device *const dev_entropy = DEVICE_DT_GET(DT_NODELABEL(rng));
```

Comment:
This could probably use a chosen entropy device instead on relying on the nodelabel being the same as for the old nrf rng.